designs/vcenter-template-import.md

```diff
@@ -44,28 +53,33 @@ manually imported the OVA into a template in vcenter, or not specify the templat
 field empty if they want it imported via a generated name that is based on the Kubernetes
 version, OS, and hash of the build. 
 
-With this command, if a user has the template name defined, we will check to see if a template of that 
-name already exists and the appropriate tags are set. If the template does not exist, we will import the template under
-that name with the appropriate tags. If the template exists but the tags don't match the bundle 
-that the CLI is configured with, we will error saying that the appropriate tags are not set, allowing
-the user to decide if they want to manually fix the tags or delete the template first before running
-the command again. 
+If the user specifies the cluster spec as input, we will get most of the template configuration
+from there except for the name. If there is a name defined, we will throw an error mentioning
+that they have a template name defined and to remove the template name if they want to import as
+we want to get the name either from the flag. If they choose not to specify a flag for the template name,
+we will default to the name that we use in the `create` command today. This will protect the user from
+making unintended changes. 
 
-If a user does not have the template name defined, then we will do what we already do today in the `create` command, 
-which is importing the template under the name that we have predefined and setting the appropriate tags 
-for the user. 
+If the user specified the flags as input, we will get all the information from the flags themselves.
+The behavior of the import would not change otherwise, where they can choose to specify the template 
+name or not. 
 
 For both cases, we will have output saying that the template has been imported under the 
 specific name set in vcenter. If the user did not have the template name defined, the `create`
 cluster will still use the template that was imported before because the name would match.
 It would only change if the user ran `import`, upgraded the CLI (or the bundle got updated), and then 
-ran `create`.
+ran `create`. The template would also be imported with the appropriate tags with any of the options.
+
+If the user specifies both cluster spec and flags, we will throw an error mentioning to choose
```

Comment:
Maybe we could just say here than flags take precedence over values in cluster spec, instead of throwing an error?